File: python/gigasecond/gigasecond.py
from datetime import timedelta

def add_gigasecond(birth_date):
    '''
    Calculate the moment when someone has lived for 10^9 seconds.
    
    Input: birth_date (datetime): date of birth of person to calculate
    
    Returns: point when person has lived for a gigasecond (datetime)
    
    '''
    
    GIGASECOND = 1000000000
       
    return birth_date + timedelta(seconds=GIGASECOND)

File: javascript/gigasecond/gigasecond.js
/** Input: a datetime object.
 *  Output: a new datetime set to one gigasecond after the input datetime. */
export const gigasecond = (datetime) => {

  // Datetimes are in milliseconds.
  return new Date(datetime.getTime() + GIGASECOND * 1e3)
};

const GIGASECOND = 1e9
